portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,55 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+.topnav {
+  background-color: #333;
+  overflow: hidden;
+  margin: 0;
+  font-family: "Trebuchet MS", Helvetica, sans-serif;
+  z-index: 9;
 }
-
-#greeting-container {
-  margin-top: 20px;
+.topnav a {
+  float: left;
+  display: block;
+  color: white;
+  text-align: cetner;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 20px;
 }
+.topnav a:hover {
+  background-color: #ddd;
+  color: black;
+}
+.topnav a.active {
```

Comment:
Should this be a:active? I.e. an [active selector](https://www.w3schools.com/cssref/sel_active.asp)

Reply:
Fixed. Thanks